src/config.js

```diff
@@ -0,0 +1,5 @@
+window[Symbol.for("app-config")] = {
+  BACKEND_URL: process.env.BACKEND_URL,
+  STRIPE_API_KEY: process.env.STRIPE_API_KEY,
+  // ... other env variables
+};
```

Comment:
**Security Issue**: Storing sensitive configuration like API keys in global scope can lead to exposure and misuse of these keys. <br> **Fix**: Store sensitive keys in environment variables and access them securely within the application without exposing them to the global scope. <br> **Code Suggestion**: 
 ```
 - window[Symbol.for("app-config")] = {
 -   BACKEND_URL: process.env.BACKEND_URL,
 -   STRIPE_API_KEY: process.env.STRIPE_API_KEY,
 -   // ... other env variables
 - };
 + // Access environment variables securely within the application
 ```